code/modules/surgery/organs/external/_external_organs.dm

```diff
@@ -86,7 +86,22 @@
 	var/mutable_appearance/appearance = mutable_appearance(sprite_datum.icon, finished_icon_state, layer = -image_layer)
 	appearance.dir = image_dir
 
-	if(sprite_datum.color_src) //There are multiple flags, but only one is ever used so meh :/
+	if(ishuman(owner))
+		var/mob/living/carbon/human/human_owner = owner
+		switch(sprite_datum.color_src)
+			if(FACEHAIR)
+				appearance.color = human_owner.facial_hair_color
+			if(HAIR)
+				appearance.color = human_owner.hair_color
+			if(EYECOLOR)
+				appearance.color = human_owner.eye_color
+			if(MUTCOLORS)
+				appearance.color = image_color
+			if(MUTCOLORS2)
+				appearance.color = human_owner?.dna?.features["mcolor2"] ? human_owner.dna.features["mcolor2"] : image_color
+			if(MUTCOLORS3)
+				appearance.color = human_owner?.dna?.features["mcolor3"] ? human_owner.dna.features["mcolor3"] : image_color
```

Comment:
directly indexing dna.features is generally bad practice for limb-related code.

I'd just straight up store all MUTCOLORS on the parent `/bodypart` type. It was gonna need to happen at some point.
(A giant pr refactoring how external organs work is currently open, so I'll need to change this anyway)

Reply:
yeah that's for another refactor